Packages/io.chainsafe.web3-unity/Runtime/Scripts/CountlyAnalytics.cs

```diff
@@ -8,10 +9,12 @@ public class CountlyAnalytics : IAnalyticsClient
 {
     private const string AppKey = "4d2f30cecf1b7e2b8cd909103c1fac971872aa3f";
     private const string ServerUrl = "https://chainsafe-40aca7b26551e.flex.countly.com";
+    private bool analyticsOptOut = false;
```

Comment:
no need for this. We already store the projectconfig in the countly analytics. so just check the projectConfig.AnalyticsOptOut in CaptureEvent

Reply:
yep